src/server/utilities/cspHeader/index.js

```diff
@@ -9,14 +9,66 @@ import getOriginContext from '#contexts/RequestContext/getOriginContext';
  * View the developer console for errors.
  */
 
+const advertisingDirectives = {
```

Comment:
I think as a follow on to this we could start looking at creating a different CSP for different page types? We don't need these for all pages, we don't need includes for all pages etc, there's potentially more cases like this?

Reply:
That sounds like a good idea 👍